css/index.css

```diff
@@ -21,13 +21,49 @@ time, mark, audio, video {
 	border: 0;
 	font-size: 100%;
 	font: inherit;
-	vertical-align: baseline;
+    vertical-align: baseline;
+    text-decoration: none;
 }
 /* HTML5 display-role reset for older browsers */
 article, aside, details, figcaption, figure, 
 footer, header, hgroup, menu, nav, section {
 	display: block;
 }
+
+a {
+    color: black;
+}
+
+.top_width {
```

Comment:
With the markup for the header I showed above you can now style your header/nav as such

```css
header{
    padding-bottom:4%;
}
nav{
    display:flex;
    justify-content:space-between;
    align-items:center;
    padding:4% 0;
}
nav a{
    color:#000;
    text-decoration:none;
}
```